projects/Test/Integration/TestExchangeDeclare.cs

```diff
@@ -61,18 +61,21 @@ public async Task TestConcurrentExchangeDeclareAndBind()
             {
                 async Task f()
                 {
-                    try
+                    using (IChannel ch = await _conn.CreateChannelAsync())
```

Comment:
Wasn't the idea of these tests to verify that they work when called in one channel? So splitting them to many channels defeats the purpose of those tests no?

Reply:
See my comment [here](https://github.com/rabbitmq/rabbitmq-dotnet-client/pull/1233#discussion_r1608906517).

`IModel` / `IChannel` has never been guaranteed to be thread safe, so I removed

Reply:
Now that `Dispose` does not use `GetAwaiter().GetResult()`, I can modify these tests to test both cases.